### ISSUE TITLE
[Feature request] Display a raster in Margin Step

### ISSUE THREAD

**Issue by gorgobacka** [4lex4/scantailor-advanced#130]
Usually, I try to adjust the content based on constant objects (mostly the page numbers). Guides are a great way to do this. But often, I have problems to adjust the content of the left and right page with identical distances.

I expect a raster overlay can improve the content alignment.

**Comment by zvezdochiot** [comment]
Repeat! See #93, #79.

**Comment by gorgobacka** [comment]
I don't think it repeats the other issues. The use case is the same. But the proposed solution is different: a raster overlay.

Is your thumb down for displaying a raster?

**Comment by Piolie** [comment]
What is a raster overlay? And how would it help with page alignment?

**Comment by 4lex4** [comment]
If you create a feature request, describe the feature and how that should work in details, so that other people could understand it, and explain what it's for by presenting some practical use case(s) of the proposed feature.

**Comment by gorgobacka** [comment]
Let me try to explain it more in detail.

The idea is to display a raster to support the margin step. You can use the lines to adjust the content of pages. Specially, in cases where the "Alignment" functionality doesn't work well. E.g. to get the identical distance of the page number to the page border for left and right pages. (Guides lack the possibility to add a second guide that is identical to the first but on other site of the page: e.g. guideA: 1 inch to the left, guideB: 1 inch to the right.)

Here is a possible visualisation of the requested feature:
![example](https://user-images.githubusercontent.com/10488441/82764079-997b1d00-9e0c-11ea-93bc-28d3176ab08a.png)

On the left site there can be a new menu, to adjust the distances of the lines within the raster. The raster should be displayed below the margin rectangles and the centre of the raster is the centre of the final page size.

I can think of some further additions:
- Add guide according a line in the raster.
- Attach the content automatically to the nearest line of the raster.

**Comment by Piolie** [comment]
Now I get it. You are proposing what most of us would call a _grid_.

I personally wouldn't find it very useful. I think that the best way to achieve perfect margin distances would be guides that can be set with specific distances, plus some anchor points to align to those lines, as discussed in #79.

**Comment by gorgobacka** [comment]
> Now I get it. You are proposing what most of us would call a _grid_.

That is indeed the correct word. I changed the title.

> I personally wouldn't find it very useful. I think that the best way to achieve perfect margin distances would be guides that can be set with specific distances, plus some anchor points to align to those lines, as discussed in #79.

"Guides that can be set with specific distances" would be also fine for me. But at the moment, I don't have a good idea how the UI components could look like to archive this.

**Comment by Piolie** [comment]
> I don't have a good idea how the UI components could look like to archive this.

Maybe `right click` on the guide -> `Set distance to page border`. A list of the current guides with editable distances on the space where you suggested the raster menu is another option.

**Comment by gorgobacka** [comment]
@4lex4 Do you have any opinions about the proposed ideas to specify the positions of guides?

Currently, I'm holding a ruler close to the monitor to get identical distances. But I would like to have a better solution. :)
